vdr/types/common.go

```diff
@@ -53,6 +53,8 @@ type DocumentMetadata struct {
 	TimelineID hash.SHA256Hash `json:"timelineID"`
 	// Hash of DID document bytes. Is equal to payloadHash in network layer.
 	Hash hash.SHA256Hash `json:"hash"`
+	// Deactivated indicates if the document is deactivated
+	Deactivated bool `json:"deactivated"`
```

Comment:
This property is also stored which could make it deviate from the actual state. Maybe the memory store must set/unset this property which each load?

Reply:
This is on purpose. The intention was to deactivate the document. If the form of a deactivated document changes in the future, this should not change the state of an already deactivated document.